java-checks/src/test/files/checks/EnumMapCheck.java

```diff
@@ -30,5 +30,11 @@ public void compliant() {
     new HashMap<String, String>();
     int a = 5;
     a = 3;
+    Map<COLOR, String> moodMapWithNullKey = new HashMap<COLOR, String>(COLOR.class);  // compliant because of next line.
```

Comment:
This line won't compile, there is no such constructor for `HashMap`. Also comment is saying "next line" but I believe you mean line with `null` argument for key?